addons/l10n_ru/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2013-2018 CodUP (<http://codup.com>).
+#
+##############################################################################
+
+{
+    'name': 'Russia - Accounting',
+    'version': '3.1',
+    'summary': 'План счетов РФ',
+    'category': 'Localization',
+    'description': """
+This is the base module to manage the accounting chart for Russia in OpenERP.
+==============================================================================
+Возможности:
+
+  - План счетов бухгалтерского учёта финансово-хозяйственной деятельности организаций, утверждённый Приказом Минфина РФ от 31.10.2000 года № 94н
+    """,
+    'author': 'CodUP',
+    'website': 'http://codup.com',
+    'license': 'AGPL-3',
```

Comment:
I'm not sure that Odoo s.a. is ready to take this module with restrictions of AGPL licence. 
Also, these module is published in apps store:

https://apps.odoo.com/apps/modules/13.0/l10n_ru/
https://apps.odoo.com/apps/modules/13.0/l10n_ru_doc/

I suppose we need authors authorization to proceed

@wowas18 ?